test/gtest/common/test_ucc.cc

```diff
@@ -391,13 +399,60 @@ void proc_context_create(UccProcess_h proc, int id, ThreadAllgather *ta, bool is
     throw std::runtime_error(err_msg.str());
 }
 
+void proc_onesided_context_create(UccProcess_h proc, int id,
+                                  ThreadAllgather *ta)
+{
+    ucc_status_t         status;
+    ucc_context_config_h ctx_config;
+    std::stringstream    err_msg;
+    ucc_mem_map_t        map[3];
+
+    status = ucc_context_config_read(proc->lib_h, NULL, &ctx_config);
+    if (status != UCC_OK) {
+        err_msg << "ucc_context_config_read failed";
+        goto exit_err;
+    }
+    for (auto i = 0; i < 3; i++) {
```

Comment:
i don't like 3 and 1<<20 being hardcoded and used in multiple places. Lets define macros: UCC_TEST_N_MEM_SEGMENTS, UCC_TEST_MEM_SEGMENT_SIZE

Reply:
done